zuul/connection/github.py

```diff
@@ -165,6 +165,26 @@ def _event_issue_comment(self, request):
         event.type = 'pr-comment'
         return event
 
+    def _event_pull_request_review(self, request):
+        """Handles pull request reviews"""
+        body = request.json_body
+        action = body.get('action')
+        if action != 'submitted':
+            return
+        pr_body = body.get('pull_request')
+        if pr_body is None:
+            return
+
+        review = body.get('review')
+        if review is None:
+            return
+
+        event = self._pull_request_to_event(pr_body)
+        event.state = body.get('review').get('state')
```

Comment:
line 178 already get()s the review, could avoid the extra dict lookup here by referencing 'review'